pkgs/development/libraries/dbus/default.nix

```diff
@@ -60,7 +61,8 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs =
-    lib.optionals x11Support [
+    [ libapparmor ]
```

Comment:
we seem to be getting an eval issue due to this being added unconditionally 
```
Failed to evaluate libapparmor-2.13.4: «unsupported»: is not supported on ‘x86_64-darwin’
```

you should probably do `!stdenv.isDarwin`